lib/cli/output/eval.js

```diff
@@ -37,9 +41,9 @@ const printTableAndGetScore = (data) => {
 };
 
 const printStatusAndLog = (data, totalScore) => {
-  logger.moduleLog('debug', 'Eval Output', `Evaluation logs: ${new Buffer(data.log, 'base64').toString()}.`);
+  logger.moduleLog('debug', 'Eval Output', `Evaluation logs: ${Buffer.from(data.log, 'base64').toString()}.`);
```

Comment:
?

Reply:
The Buffer constructor is different depending on the type of its argument, and not validating such arguments might lead to memory disclosure and denial of service. So eslint enforces the use of `Buffer.from()` or `Buffer.alloc()` instead. 

[ref](https://eslint.org/docs/rules/no-buffer-constructor)